lib/app-components/addon/components/project-metadata/component.ts

```diff
@@ -75,6 +79,36 @@ export default class ProjectMetadata extends Component {
         }
     }
 
+    validateCollectionLicense() {
+        return async (_: unknown, newValue: LicenseModel, oldValue: Promise<LicenseModel>, changes: Partial<Node>) => {
+            const licensesAcceptable = await this.provider.queryHasMany('licensesAcceptable', {
+                page: { size: 20 },
+            });
```

Comment:
This doesn't necessarily need to change now, but it occurs to me that this isn't very future-proofed. It would make more sense to filter the licenses on the name of the license that you're validating against and see if anything returns, rather than grabbing the first 20 licenses and hoping it's in there. It works for now, though, while we have fewer than 20 licenses.